app.js

```diff
@@ -70,7 +70,7 @@ app.locals.marked = marked;
 if (app.get('env') == 'development') {
   app.use(errorHandler());
 }
-//var token = 'SECRET_TOKEN_f8ed84e8f41e4146403dd4a6bbcea5e418d23a936';
+var token = 'SECRET_TOKEN_f8ed84e8f41e4146403dd4a6bbcea5e418d23a936';
```

Comment:
### <sub><img src="https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-high.svg" width="20"/></sub>&nbsp; Hardcoded Secret

Avoid hardcoding values that are meant to be secret. Found a hardcoded string used in here.

<h6>
<a href="https://cwe.mitre.org/data/definitions/547.html">CWE-547</a> | <a href="https://docs.snyk.io/manage-risk/prioritize-issues-for-fixing/priority-score">Priority score 810</a> | Line 73</h6>

<details>

<summary><b>Data flow: 2 steps</b></summary>

`Step 1` https://github.com/abertola123/snyk-goof/blob/5928c61e09fdceeb1fc80733a281b0db4e9ad774/app.js#L73

`Step 2` https://github.com/abertola123/snyk-goof/blob/5928c61e09fdceeb1fc80733a281b0db4e9ad774/app.js#L73

</details>